service/src/main/resources/application.yml

```diff
@@ -132,4 +125,42 @@ drshub:
       mTlsConfig:
         keyPath: rendered/ras-mtls-client.key
         certPath: rendered/ras-mtls-client.crt
+  compactIdHosts:
+    "dg.4503": staging.gen3.biodatacatalyst.nhlbi.nih.gov
```

Comment:
`dg.4503` is the BDC compact id for production only.
It should never appear or be used in preprod tiers. 
Cross-deployment-tier data access does not work in general (for Gen3 or TDR)

If this line were deleted, and DRSHub unexpectedly received a DRS URI containing `dg.4503` would this result in the "compact identifier not found" error?
If so, then I think removing this line (or replacing it with a comment) is the appropriate change.

Reply:
Great info, thanks for surfacing this! You're correct in that if this is removed from specific tiers, it should throw the "compact identifier not found" error if a URL with this id is resolved. That sounds like the correct behavior to me, I'll update the configs